source/daemon/include/daemon.h

```diff
@@ -129,11 +127,35 @@ class Daemon : public QTcpServer
     Q_OBJECT
 
 public:
-    Daemon(QString username, QString password, QString new_gpsdevname, int new_verbose, quint8 new_pcaPortMask,
-        float *new_dacThresh, float new_biasVoltage, bool bias_ON, bool new_dumpRaw, int new_baudrate,
-        bool new_configGnss, unsigned int new_eventTrigger, QString new_PeerAddress, quint16 new_PpeerPort,
-        QString new_serverAddress, quint16 new_serverPort, bool new_showout, bool new_showin, bool preamp1, bool preamp2, bool gain, QString station_ID,
-        bool new_polarity1, bool new_polarity2, QObject *parent = 0);
+    struct configuration {
```

Comment:
Very good idea to replace the endless list of constructor arguments with a configuration struct. Which config parameters would we need additionally and could one use the same struct for config file interaction? Nothing to be solved with this release, just to keep it in mind to take care of for the next one.